app/views/manifests/index.html.erb

```diff
@@ -1,7 +1,175 @@
-<h3>Recently filed manifests</h3>
+<div class="usa-width-two-thirds">
 
-<% @manifests.each do |manifest| %>
-  <li><%= manifest.tracking_number %></li>
-  <li><%= manifest.generator_name %></li>
-  <li><%= manifest.created_on %></li>
-<% end %>
+  <h3>Recently filed manifests</h3>
+
+  <table>
+    <thead>
+      <th>Tracking Number</th>
+      <th>Generator Name</th>
+      <th>Disposal Facility</th>
+      <th>Date</th>
+    </thead>
+    <tbody>
+      <% @manifests.each do |manifest| %>
+        <tr>
+          <td><%= link_to manifest.tracking_number, manifest_path(manifest) %></td>
+          <td><%= manifest.generator_name %></td>
+          <td><%= manifest.disposal_facility %></td>
+          <td><%= manifest.created_on %></td>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
+
+</div>
+
+<div class="usa-width-one-third">
+  <form class="callout">
+    <fieldset>
+      <h2>Filter manifests</h2>
+      <label class="featured" for="generator_name">Generator name</label>
+      <input id="generator_name" ng-model="data.generator.name" name="generator_name" type="text" class="ng-pristine ng-valid ng-touched">
+
+      <label class="featured" for="tsdf_name">Disposal facility name</label>
+      <input id="tsdf_name" ng-model="data.tsdf_name" name="tsdf_name" type="text" class="ng-pristine ng-untouched ng-valid">
+
+      <div class="usa-grid-full">
+        <div class="usa-width-one-half">
+          <label for="origin_state">Generator state</label>
+          <select name="origin_state" id="origin_state" class="usa-input-grid-tiny tiny-input">
+            <option value="">--</option>
+            <option value="AL">AL</option>
```

Comment:
instead of listing all of these here, we can take advantage of https://github.com/18F/e-manifest/blob/master/app/helpers/states_helper.rb

```ruby
<% us_states.each do |state| %>
 <option value=<%=state[1]%>><%=state[1]%></option>
<% end%$>
```
```